src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnection.cs

```diff
@@ -51,12 +59,25 @@ public partial class HubConnection
         public TimeSpan ServerTimeout { get; set; } = DefaultServerTimeout;
         public TimeSpan HandshakeTimeout { get; set; } = DefaultHandshakeTimeout;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HubConnection"/> class.
+        /// </summary>
+        /// <param name="connectionFactory">The connection factory.</param>
```

Comment:
If we're going to doc these, we might as well put something more useful in them:

`The <see cref="IConnectionFactory" /> used to create a connection each time <see cref="StartAsync" /> is called.`